src/gluonts/nursery/temporal_hierarchical_forecasting/model/cop_deepar/_estimator.py

```diff
@@ -208,6 +211,17 @@ def __init__(
 
         print(f"Distribution output: {base_estimator_hps['distr_output']}")
 
+        if "impute_missing_values" not in base_estimator_hps:
+            base_estimator_hps["impute_missing_values"] = impute_missing_values
+
+        if "imputation_method" not in base_estimator_hps:
+            base_estimator_hps["imputation_method"] = imputation_method
+
+        if "num_imputation_samples" not in base_estimator_hps:
+            base_estimator_hps[
```

Comment:
You can use `base_estimator_hps.setdefault("num_imputation_samples", num_imputation_samples)` instead.